data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
+		Sunbeam 4
+		"Thunderhead Launcher" 4
+		"Thunderhead Missile" 160
+		"Bullfrog Anti-Missile" 2
+		"Korath Warder"
```

Comment:
Sunbeam Shield Beetle variant's outfits seem random because they were tuned after dozens of iterations to be an unstoppable killing machine. To be as good as the sunbeam beetle, it must:

1. Go one-on-one with any human-space Korath Exile ship, or Wanderer ship, and win easily,
2. Kill a small pirate fleet without taking significant damage,
3. Survive the incoming pirates at Devil Hide without any help for a minute.

Reply:
Well those missiles will rekt all of that and we only lost 33% of beam firepower that is only useful in short range and destroys your own ship trough power usage and heat
I'll be happy to test all of this though and rekt the old beetle as well x)

Reply:
I'm experimenting with the more dangerous Korath Exile outfits.

Also, the battle is much harder than when I tested before, so there's no way any shield beetle could solo it. I can spec it to be quite strong though.

Reply:
I'll switch to the smaller Korath automata outfits. The Korath Exiles never raided the Hai, and it is unlikely the Unfettered would risk crossing automata space to search for more prey.